### DISCUSSION TITLE
Blank screen running Spectrum sample program under qemu

### DISCUSSION THREAD

**Discussion by petemoore** [rsta2/circle#504] [Q&A] [answered]
I just wanted to check if this is expected. Under QEMU 9.1.1 I am getting a blank screen running the Spectrum sample:

```
git clone git@github.com:rsta2/circle.git && cd circle && git checkout develop && ./configure -r 3 -p aarch64-none-elf- --qemu && ./makeall && cd addon/Spectrum && make && cd sample && make && qemu-system-aarch64 -full-screen -M raspi3b -kernel kernel8.img -serial null -serial stdio
```

I'm away from my pis to check on a real device. Many thanks!

**Comment by rsta2** [comment, marked as answer]
The *addon/Spectrum/sample* uses a special 4-bits depth mode for the frame buffer, which is not supported by QEMU, how it seems. The sample does work on a real RPi Zero, I checked it.

**Reply by petemoore** [reply]
Many thanks Rene!
